src/essence/Tools/Isochrone/models/Model.js

```diff
@@ -0,0 +1,16 @@
+export default class Model {
```

Comment:
so if you wanted to create a new cost model, this is the template you would use?